doc/CONTRIBUTING.md

```diff
@@ -337,6 +337,34 @@ for more configuration details. Everything within `python_config` is imported
 via `dictConfig` (in addition to some standard settings defined in
 `dataactcore.logging`.
 
+### Setup with Docker
+
+Install docker in your local machine by selecting your OS and hitting install from this [link](https://docs.docker.com/docker-for-mac/install).
+This [link](https://docs.docker.com/docker-for-mac/install/) will already include docker-compose.
+
+These are the files you need to add/change before you begin working in your container. `_example` or `example_` needs to be removed from these file names.
```

Comment:
We probably want to say you want to make copies and remove the text there since we don't want them removing the example files entirely if they're ever going to make a commit (or if this is a dev that's joining us)